core-bundle/contao/templates/forms/form_checkbox.html5

```diff
@@ -34,5 +34,9 @@
       <?php endif; ?>
     <?php endforeach; ?>
 
+    <?php if ($this->help): ?>
+      <span class="help" id="help_ctrl_<?= $this->id ?>"><?= $this->help ?></span>
```

Comment:
I think a `<p>` tag would be semantically more correct here (and in the other templates), don't you think?